app/src/components/submissionPage/CharactersLeftDisplay.tsx

```diff
@@ -1,6 +1,18 @@
 import MentionElements from "../../types/MentionElements";
 import GetExtraLength from "./GetExtraLength";
 import React from "react";
+import styled from "styled-components";
+
+const HelperText = styled.p`
+    text-align: right;
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    color: #faf8f8;
+    margin-right: 10px;
+`;
```

Comment:
This is a near duplicate of the `HelperText` styled used in `SubmissionTextBox`. It is possible to [extend styles](https://styled-components.com/docs/basics#extending-styles), which would help remove duplication.